### PR TITLE
Code base cleaned up (deprecated and deleted)

### LINKED ISSUE #861: Revit_Toolkit: remove deprecated 3.0 and 3.1 code base
#### Description:

As in the title.

### LINKED ISSUE #862: Revit_Toolkit: Deprecate BH.Engine.Adapters.Revit.Plane method
#### Description:

It is not used anywhere and does not fit the purpose of the engine, therefore should be deprecated.

### PR DESCRIPTION
**Pull request by pawelbaran**
### Issues addressed by this PR

Closes #861
Closes #862

### Test files

Not needed, only deprecation and deletion of deprecated code base done in this PR. Checking if compiles should be enough 😉 

### Changelog

- `BH.Engine.Adapters.Revit.Query.Plane` method has been deprecated
- Deprecated code base older than 3.2 has been deleted

### Additional comments

**-1184 net!** 🥳

### CODE REVIEW

File: Revit_Engine/Query/Plane.cs [hunk 1]
```diff
@@ -34,6 +34,7 @@ public static partial class Query
         /****              Public methods               ****/
         /***************************************************/
 
+        [Deprecated("4.0", "BH.Engine.Adapters.Revit.Query.Plane method is not used any more.")]
```
Comment (IsakNaslundBh) [inline Revit_Engine/Query/Plane.cs:37]: The deprecated attribute is actually... Deprecated.

image


If this is to simply be deleted, should be using the `ToBeRemovedAttribute` instead, if replaced by something else, versioning should generally be the first goto, if not at all possible, the `ReplacedAttribute` should be used.

Comment (pawelbaran) [reply Revit_Engine/Query/Plane.cs]: Depr... fixed ;-)

**Review by LMarkowski — APPROVED** [review]
LGTM 👍

**Review by IsakNaslundBh — APPROVED** [review]
LGTM

**Pull request merged**